scripts/package_sign_tacos.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
```

Comment:
Does this need to be bash or could it be a bourne shell?

Reply:
Removed line that forced bash shell.